src/tests/async/PGO.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
All other test files in this directory are lower-case. Can you please make this match?